### ISSUE TITLE
Add missing frontend functions to compile popular models

### ISSUE THREAD

**Issue by guillesanbri** [unifyai/ivy#9561]
This list holds some frontend functions used in popular models that are currently missing. When adding new function to this ToDo list do so in alphabetical order (within each framework) :slightly_smiling_face: 

# Torch
- [ ] torch.functional.F.embedding (alias to one function in the HF ToDo list)
- [ ] torch.functional.F.max_pool2d
- [ ] torch.Tensor.reshape_as

# TF
- [ ] tensorflow.compat.v1.nn.fused_batch_norm (alias to one function in the HF ToDo list)
- [ ] tensorflow.compat.v1.pad
- [ ] tensorflow.math.ceil
- [ ] tensorflow.matmul
- [ ] tensorflow.nn.softmax

# JAX

**Comment by Ishticode** [comment]
- [ ] #9706

**Comment by Ishticode** [comment]
- [ ] #9707

**Comment by MahmoudAshraf97** [comment]
#10523

**Comment by MuhammadSaeedBatikh** [comment]
#10599

**Comment by paulaehab** [comment]
- [ ] #14816

**Comment by paulaehab** [comment]
- [ ] #14946

**Comment by paulaehab** [comment]
- [ ] #15048
